### DISCUSSION TITLE
JWT Token

### DISCUSSION THREAD

**Discussion by imknight** [supabase/supabase#4519] [Questions] [answered]
decode the JWT Token and find the exp parameters, but to convert this exp to proper date? need to find out when the token will be expired.

**Comment by b2m9** [comment, marked as answer]
Not sure which platform you're using, but I assume JS. In general, you need to know that JWT by standard expresses timestamps **in seconds, not milliseconds**. Multiply it by 1000, then you can use the timestamp to create a Date in JS for example: `new Date(session.expires_at * 1000)`

**Reply by imknight** [reply]
thanks
